### PR TITLE
Create a new screen/page: "Add network details/confirmation" #12899

### LINKED ISSUE #12899: Create a new screen/page: "Add network details/confirmation"
**User Story** 
As a user I want to be able to add a popular custom network from the custom network switcher modal.
Figma link: https://www.figma.com/file/wfOwLmE52OKY0bSrsQwHPL/Add-popular-custom-network?node-id=1%3A3

**Task(s)**

- Create a new page: "Add network details/confirmation"

image

Acceptance criteria:
- The top of the page should show the network name and symbol
- Info icon: "MetaMask does not verify any networks" should open an info popup as shown in the screenshot:
image
  
- Learn about scams and network security risks is a link to the external page
- "View details" option,  should show all the details in the modal pop-up window, like it is currently implemented. Screenshots below:
image
image

### PR DESCRIPTION
**Pull request by dragana8**
Fixes: #12899

Explanation:  
Storybook file for https://github.com/MetaMask/metamask-extension/issues/12899
Screenshot from 2021-12-16 11-33-14

### CODE REVIEW

**Comment by mirjanaKukic** [comment]
Verified by QA.

**Review by david0xd — DISMISSED** [review]

File: app/_locales/en/messages.json [hunk 1]
```diff
@@ -1487,6 +1490,9 @@
   "learmMoreAboutGas": {
     "message": "Want to $1 about gas?"
   },
+  "learnAbout": {
+    "message": "Learn about"
```
Comment (darkwing) [inline app/_locales/en/messages.json:1494]: For these types of strings, where they feature a link for part of them, we set the string to:

```Learn about $1```

The for the JSX, we do something like:

```
t('textKey1', [
        <a
          href=""
          className="network-status__tooltip-label"
        >
          {t('textKey2')}
        </a> ])
```

File: ui/components/ui/network-details/index.scss [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+    background: #f2f3f4;
+    border-radius: 16px;
+    margin-left: 63px;
+    margin-top: 24px;
+    font-size: 12px;
+    padding-top: 7px;
+    justify-content: center;
+
+    & .identicon__image-border {
+      margin-right: 8px;
```
Comment (darkwing) [inline ui/components/ui/network-details/index.scss:27]: `margin-inline-end` might be better for RTL

File: ui/components/ui/network-details/index.scss [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+    }
+  }
+
+  &__header {
+    display: flex;
+    width: 175px;
+    height: 32px;
+    background: #f2f3f4;
+    border-radius: 16px;
+    margin-left: 63px;
```
Comment (darkwing) [inline ui/components/ui/network-details/index.scss:20]: `margin-inline-start` might be better for RTL

**Review by darkwing — APPROVED** [review]

**Review by darkwing — CHANGES_REQUESTED** [review]
Didn't mean to approve.

File: app/_locales/en/messages.json (hunk 1 above)
Comment (dragana8) [reply app/_locales/en/messages.json]: updated

File: ui/components/ui/network-details/index.scss (hunk 2 above)
Comment (dragana8) [reply ui/components/ui/network-details/index.scss]: updated

File: ui/components/ui/network-details/index.scss (hunk 3 above)
Comment (dragana8) [reply ui/components/ui/network-details/index.scss]: updated

File: ui/components/ui/network-details/index.scss [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+    }
+  }
+
+  &__footer {
+    padding: 16px;
+    height: 70px;
+  }
+
+  &__cancel {
+    margin-right: 16px;
```
Comment (darkwing) [inline ui/components/ui/network-details/index.scss:37]: `margin-inline-end`

File: ui/components/ui/network-details/index.scss [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+    font-weight: 600;
+  }
+
+  &__subtitle {
+    @include H7;
+
+    display: flex;
+    align-items: center;
+    text-align: center;
+    margin-left: 5px;
```
Comment (darkwing) [inline ui/components/ui/network-details/index.scss:57]: `margin-inline-start`

File: ui/components/ui/network-details/index.scss [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+      flex-direction: row;
+      margin-left: 19px;
+
+      &__link {
+        padding: 0;
+        font-size: 12px;
+        color: #037dd6;
+        border: 1px solid transparent;
+        background-color: transparent;
+        margin-right: 18px;
```
Comment (darkwing) [inline ui/components/ui/network-details/index.scss:75]: margin-inline-end

File: ui/components/ui/network-details/index.scss [hunk 7]
```diff
@@ -0,0 +1,154 @@
...
+  }
+
+  &__info {
+    padding-top: 24px;
+    font-size: 12px;
+
+    &__link-section {
+      display: flex;
+      flex-direction: row;
+      margin-left: 19px;
```
Comment (darkwing) [inline ui/components/ui/network-details/index.scss:67]: margin-inline-start

File: ui/components/ui/network-details/index.scss [hunk 8]
```diff
@@ -0,0 +1,154 @@
...
+    }
+
+    &__icon {
+      width: 10px;
+    }
+
+    &__bold-label {
+      font-weight: 600;
+      display: flex;
+      margin-left: 24px;
```
Comment (darkwing) [inline ui/components/ui/network-details/index.scss:91]: margin-inline-start

File: ui/components/ui/network-details/index.scss [hunk 9]
```diff
@@ -0,0 +1,154 @@
...
+      color: #24292e;
+
+      & .fa-info-circle {
+        color: #b8b8b8;
+        width: 10px;
+        height: 10px;
+        margin-left: 4px;
```
Comment (darkwing) [inline ui/components/ui/network-details/index.scss:98]: margin-inline-start

File: ui/components/ui/network-details/index.scss [hunk 10]
```diff
@@ -0,0 +1,154 @@
...
+  }
+
+  &__content {
+    margin: 32px 24px 30px 24px;
+    width: 248px;
+    height: 170px;
+    border: 1px solid #d6d9dc;
+    border-radius: 9px;
+    position: relative;
+    padding-left: 16px;
```
Comment (darkwing) [inline ui/components/ui/network-details/index.scss:110]: padding-inline-start

File: ui/components/ui/network-details/network-details.component.js [hunk 11]
```diff
@@ -0,0 +1,114 @@
...
+          {t('doesNotVerify')}
+          <Tooltip position="bottom" theme="light" title={t('scamsAndNetwork')}>
+            <i className="fa fa-info-circle" />
+          </Tooltip>
+        </label>
+        <div className="network-details__info__link-section">
+          <I18nValue
+            messageKey="learnAbout"
+            options={[
+              <a href="" key="learnAboutLnk">
```
Comment (darkwing) [inline ui/components/ui/network-details/network-details.component.js:73]: Do we have a URL for this?

**Review by darkwing — CHANGES_REQUESTED** [review]

File: ui/components/ui/network-details/network-details.component.js (hunk 11 above)
Comment (dragana8) [reply ui/components/ui/network-details/network-details.component.js:73]: Not yet.

File: ui/components/ui/network-details/index.scss (hunk 4 above)
Comment (dragana8) [reply ui/components/ui/network-details/index.scss]: updated

File: ui/components/ui/network-details/index.scss (hunk 6 above)
Comment (dragana8) [reply ui/components/ui/network-details/index.scss:75]: updated

File: ui/components/ui/network-details/index.scss (hunk 7 above)
Comment (dragana8) [reply ui/components/ui/network-details/index.scss:67]: updated

File: ui/components/ui/network-details/index.scss (hunk 9 above)
Comment (dragana8) [reply ui/components/ui/network-details/index.scss]: updated

File: ui/components/ui/network-details/index.scss (hunk 10 above)
Comment (dragana8) [reply ui/components/ui/network-details/index.scss]: updated

**Comment by darkwing** [comment]
Marking as DNM until we have the required link.

File: ui/components/ui/network-details/network-details.component.js (hunk 11 above)
Comment (darkwing) [reply ui/components/ui/network-details/network-details.component.js:73]: @dragana8 Do you know who to go to get this link?  @danjm do you know?

Comment (bschorchit) [reply ui/components/ui/network-details/network-details.component.js:73]: Here's the link for it: https://metamask.zendesk.com/hc/en-us/articles/4404424659995

Comment (dragana8) [reply ui/components/ui/network-details/network-details.component.js:73]: @darkwing we added the link.

**Comment by brad-decker** [comment]
@bschorchit @dragana8 how is this page different from the add network confirmation? There's a lot of work that went into this was redundant -- all of the components for this view already exist and the entire page itself is ...very similar to the confirmation. Can we discuss reusing what we have before this goes in?

**Comment by bschorchit** [comment]
@EHaracic could the team work on this #13400 so we can move forward with unblocking this PR? The goal of this ticket is to record a video so the designers understand what it would look like with those changes and make some design decisions on it. Thank you!

File: app/_locales/en/messages.json (hunk 1 above)
Comment (Nunnaphat1234) [reply app/_locales/en/messages.json]: สวัสดีค่ะ เฮ้อ_เหนื่อย
หนูกำลังติดต่อ_ครูสอนทางฝั่งไทยอยู่นะ

แชร์Code ให้ (ความลับ)ตรงไกนได้บ้าง

>

**Comment by Nunnaphat1234** [comment]
https://photos.app.goo.gl/vwoibCuiTVHHtoow6


Thank you very much to the team. Look at them as my contract. Roll the bag
on it. With that there, how will you coordinate? Can I have Twitter? I just
knew that I could share the code. I have to go to work Anyway, it's dark
again.

ในวันที่ ศ. 1 ก.ค. 2022 13:38 น. b_ลีลๅ_d ***@***.***>
เขียนว่า:

**Review by Nunnaphat1234 — COMMENTED** [review]
Yes

File: app/_locales/en/messages.json (hunk 1 above)
Comment (Nunnaphat1234) [reply app/_locales/en/messages.json]: Yes, about the gas cost. I want matic go to polagon chain การ more than that I sent wrong 555

Comment (Nunnaphat1234) [reply app/_locales/en/messages.json]: https://photos.app.goo.gl/Hn1X9jmT42YUxvNJ7

File: ui/components/ui/network-details/index.scss (hunk 3 above)
Comment (Nunnaphat1234) [reply ui/components/ui/network-details/index.scss]: Set up KitHub in Thai language as well.

**Comment by brad-decker** [comment]
@bschorchit @dragana8 what is the status of this PR? Should we close this as I thought the add networks flow was done?

**Comment by dragana8** [comment]
@brad-decker I think it can be closed.

**Pull request closed without merging**